tests/src/test/scala/pathy/path.scala

```diff
@@ -100,133 +84,194 @@ class PathSpecs extends Specification with ScalaCheck {
     ).toList must_== List("c", "foo", "bar", "flat.md")
   }
 
-  "parseRelFile - image.png" in {
-    parseRelFile("image.png") must beSome(file("image.png"))
-  }
+  "parseRelFile" >> {
```

Comment:
Agreed. Although I think some of them are meant to be general. For those I could make an `Arbitrary[PathCodec]` and make them properties. Unfortunately, `scalacheck` is not smart enough to realize that there are only a finite amount of `PathCodec` and only test them once... *sigh*